packages/docs/nuxt.config.ts

```diff
@@ -107,6 +107,7 @@ export default defineNuxtConfig({
       },
       'Material+Icons': true,
       'Material+Icons+Outlined': true,
```

Comment:
Might worth it to remove old Material Icons then. And check docs for regression.

Reply:
removed